cluster/addons/dns/kubedns-sa.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
If this need to be created along with the addon, may need to add label `kubernetes.io/cluster-service: "true"`. Otherwise addon-manager will omit this.